operations/enterprises.md

```diff
@@ -726,6 +732,11 @@ Adds a new company to the enterprise.
 | `BillingCode` | string | optional | Billing code of the company. |
 | `AccountingCode` | string | optional | Accounting code of the company. |
 | `Address` | [Address](configuration.md#address) | optional | Address of the company. |
```

Comment:
Merge the master, the `configuration.md#address` has changed to `configuration.md#address-parameters`